### ISSUE TITLE
[Feature]: Create a react native version of the Celo Composer

### ISSUE THREAD

**Issue by GigaHierz** [celo-org/celo-composer#303]
### Tell us about your idea

The old React Native version of the Celo Composer has been removed as it was not maintained and the Celo Composer received a bigger update.

- [ ] consider the proposal from #245 to build one version for React and React Native
- [ ]  create a React Native version of the Celo Composer

**Comment by Philix27** [comment]
Will a temlate of React Native + Thirdweb be okay? 

Currently the best way to work with React Native is to use either Thirdweb or Reown Appkit/Walletkit.

**Comment by GigaHierz** [comment]
hi @Philix27 , 

thirdweb is a great choice

**Comment by Philix27** [comment]
Can this be assigned to me? @GigaHierz

**Comment by Gracing47** [comment]
is this issue still to solve?

**Comment by Philix27** [comment]
@Gracing47  
Yes, I'm working on it

**Comment by Gracing47** [comment]
So mo need for me to start? @Philix27

**Comment by Philix27** [comment]
Yes, I am waiting for @viral-sangani  to create a seperate repository for React native so that I can push my implementation there.

**Comment by GigaHierz** [comment]
Hi @philix27 , 

the repository has been created. Please create your PR there and make sure that your submission follows the guide in the issue: https://github.com/celo-org/celo-composer-react-native/issues
